velox/functions/prestosql/Probability.h

```diff
@@ -43,5 +43,22 @@ struct BetaCDFFunction {
   }
 };
 
+template <typename T>
+struct NormalCDFFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  // Normal cumulative distribution is computed as per the reference at
+  // https://mathworld.wolfram.com/NormalDistribution.html.
+
+  const double kSqrtOfTwo = sqrt(2);
```

Comment:
Its okay to move this closer to the logic to line 58 also.

Reply:
@aditi-pandit wouldn't move it to line 58 be same as what I had done at the beginning? i.e.
`   result = 0.5 * (1 + erf((value - m) / (sd *sqrt(2))));`

 If the current change I have done is okay too - can we merge this PR?

Reply:
Hi @poojabhandari, I was suggesting :
template <typename T>
struct NormalCDFFunction {
  VELOX_DEFINE_FUNCTION_TYPES(T);

  // Normal cumulative distribution is computed as per the reference at
  // https://mathworld.wolfram.com/NormalDistribution.html.
  
  FOLLY_ALWAYS_INLINE void
  call(double& result, double m, double sd, double value) {
    VELOX_USER_CHECK_GT(sd, 0, "standardDeviation must be > 0");

    static const double kSqrtOfTwo = sqrt(2);
    result = 0.5 * (1 + erf((value - m) / (sd * kSqrtOfTwo)));
  }
};

Reply:
@aditi-pandit @Yuhta updated PR. Please check